txnsync/exchange.go

```diff
@@ -73,3 +75,18 @@ type timingParams struct {
 	AcceptedMsgSeq      []uint64 `codec:"a,allocbound=maxAcceptedMsgSeq"`
 	NextMsgMinDelay     uint64   `codec:"m"`
 }
+
+const (
+	noProposal byte = iota
+	transactionsForProposal
+	proposalByte
+	proposalFilter
```

Comment:
I don't think we'll need proposalByte/proposalFilter since these are implied from the data itself.

Reply:
you mean we need exactly one of these right?

Reply:
yes, we need the `transactionsForProposal` for now.

Reply:
oh sorry, misunderstood